hawc/apps/assessment/migrations/0036_assessment_enable_downloads_and_more.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 4.2.1 on 2023-05-11 01:25
+
+from datetime import timedelta
+
+from django.db import migrations, models
+from django.db.models import Count
+from django.utils.timezone import now
+
+
+def fwd(apps, schema_editor):
```

Comment:
this sets older assessments to turn on/off the sidebar options, depending on if they have data in these comains